scribble-lib/scriblib/autobib.tex

```diff
@@ -8,3 +8,7 @@
 \newcommand{\Autocolbibentry}[1]{\setlength{\ABcollength}{\linewidth-5ex}\parbox[t]{\ABcollength}{#1\vspace{1.0ex}}}
 
 \newcommand{\Autobibref}[1]{#1}
+
+\makeatletter
+\@ifundefined{AutobibLink}{\newcommand{\AutobibLink}[1]{#1}}{}
+\makeatother
```

Comment:
Could this be `\providecommand{\AutobibLink}....` ?

Reply:
Apparently so. (I had never heard of `\providecommand`.)